client/extensions/woocommerce/state/sites/settings/general/reducer.js

```diff
@@ -53,6 +54,10 @@ export default createReducer( null, {
 		return LOADING;
 	},
 
+	[ WOOCOMMERCE_SETTINGS_GENERAL_REQUEST_FAILURE ]: () => {
```

Comment:
We need this to remove the LOADING constant that gets pushed into state of by WOOCOMMERCE_SETTINGS_GENERAL_REQUEST